### PR TITLE
Change perl6 to raku

### PR DESCRIPTION
**Pull request by sumanstats**

### CODE REVIEW

**Comment by sjn** [comment]
A whole lot of these are historical documents. Should we really change the names from Perl 6 to Raku there?

**Comment by JJ** [comment]
No, we simply shouldn't, unless it's to unbreak a broken link or something like that.

File: source/archive/doc/design/apo/A01.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <div id="archive-notice">
     <h3>This file is part of the Perl 6 Archive</h3>
     <table bgcolor="red"><tr><td><font color="white"> Note: these documents may be out of date. Do <b>not</b> use as reference!</font></tr></td></table>
-    <p>To see what is currently happening visit <a href="http://www.perl6.org/">http://www.perl6.org/</a></p>
+    <p>To see what is currently happening visit <a href="http://www.raku.org/">http://www.raku.org/</a></p>
```
Comment (JJ) [inline source/archive/doc/design/apo/A01.html:5]: It's probably no at bad idea to change these... Even if they are historical.

File: source/archive/doc/design/exe/E04.html [hunk 2]
```diff
@@ -713,10 +713,10 @@ <h3>This file is part of the Perl 6 Archive</h3>
 <p>except that its parameter list doesn't require parentheses. That
 implies:</p>
 <ul>
-<li><a name='The Perl 6 for, while, if, and given statements each take two arguments: an expression that controls them and a subroutine/closure that they execute. Normally, that closure is just a block (in Perl6 all blocks are really closures):'></a>The Perl 6 <code>for</code>, <code>while</code>, <code>if</code>, and <code>given</code> statements each take
+<li><a name='The Perl 6 for, while, if, and given statements each take two arguments: an expression that controls them and a subroutine/closure that they execute. Normally, that closure is just a block (in Raku all blocks are really closures):'></a>The Perl 6 <code>for</code>, <code>while</code>, <code>if</code>, and <code>given</code> statements each take
 two arguments: an expression that controls them and a
 subroutine/closure that they execute. Normally, that closure is just a
-block (in Perl6 <i>all</i> blocks are really closures):</li>
+block (in Raku <i>all</i> blocks are really closures):</li>
```
Comment (JJ) [inline source/archive/doc/design/exe/E04.html:719]: Most probably not here, even less so if you don't do it consistently, changing only Perl6 but not Perl 6

File: source/archive/rfc/111.pod [hunk 3]
```diff
@@ -24,7 +24,7 @@ this up.
 
 =head1 DESCRIPTION
 
-Perl6 should ignore any whitespace before the terminator of a heredoc on any
+Raku should ignore any whitespace before the terminator of a heredoc on any
```
Comment (JJ) [inline source/archive/rfc/111.pod:27]: Definitely, not this one. These are RFCs, and should stay just the way they were created by their authors.

File: source/about/index.html [hunk 4]
```diff
@@ -14,7 +14,7 @@ <h2>About raku.org</h2>
   <div class="col-sm-6">
     <div class="panel panel-default">
       <div class="panel-heading">
-        <h3 class="panel-title">Contributing to raku.org</h3>
+        <h3 class="panel-title">Contributing to perl6.org</h3>
```
Comment (JJ) [inline source/about/index.html]: These are exactly the ones that should have been changed...

**Comment by sumanstats** [comment]
Just reverted the change in historical documents.

The link to **Roast** test suite in footer  needs change. This commit https://github.com/Raku/raku.org/pull/152/commits/e49011750511c0b517fbb774bd97c8ea2532b9d5 reflects that.

**Comment by JJ** [comment]
@sumanstats that one is automatically redirected for the duration, I think, since we transferred it...

**Pull request merged**